component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mepin/MepinTransactions.java

```diff
@@ -256,23 +287,36 @@ protected String getTransaction(String url, String transactionId, String clientI
      * @return the user information
      * @throws AuthenticationFailedException
      */

```

Comment:
Change the method level comments according to the changes in getUserInformation

Reply:
fixed with 8b37d635bec44c8d19a6cebc6fed7eaaec0ff81c